test/third_party/install.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env sh
```

Comment:
I thought of that possiblity after I commented.

I'm okay if the install.sh stays here, but I'm not sure run.sh makes sense.  Run third_party? what does that entail?